source/index.md

```diff
@@ -1,4 +1,10 @@
-[![OpenFisca logo](https://openfisca.org/img/logo-openfisca.svg)](https://openfisca.org)
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" crossorigin="anonymous">
+<div class="button">
+  <a class="button_link" href="https://github.com/openfisca/openfisca-core"><i class="fab fa-github"></i> GitHub</a>
```

Comment:
I think our conclusion during the planification was that a link towards https://github.com/openfisca would be best

Reply:
Done.